### DISCUSSION TITLE
Does Primeng  Has Gantt Chart ??

### DISCUSSION THREAD

**Discussion by chinna-ul** [primefaces/community#1352] [PrimeNG] [answered]
Does PrimeNg has  Gantt Chart ??

**Comment by nckirik** [comment]
apparently it's planned for Q4 2024 according to [roadmap](https://primeng.org/roadmap)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi, we don't have gannt chart yet but it's in our roadmap.
